src/components/ArtistList.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default ArtistList
+
+ArtistList.propTypes = {
+  artists: PropTypes.array,
```

Comment:
Is this PropType required? If so, then it should be: 

`artists: PropTypes.array.isRequired,`

Reply:
Then again, in line 14 it seems a potential error / missing array is handled. So the above comment is probably obsolete.